frontend/src/pages/LearningCourse/Material/Material.tsx

```diff
@@ -1,19 +1,20 @@
 import { FC } from 'react';
 
-import { MATERIAL } from 'constants/materials';
+import { ContentElementType } from 'types/course';
 import { playVideo } from 'icons';
+import { PPViewer } from 'components/PPViewer';
 
 import { MaterialWrapper, MaterialText, MaterialVideo } from './styled';
 
 interface IProps {
   material: string;
-  materialType: string;
+  materialType: ContentElementType;
   videoPreview: string | boolean;
```

Comment:
Do we really need this type for videoPreview? It's kind of weird to have a string or boolean as a prop type.

Reply:
hey, it's not my code actually. idk what are we using this prop for